DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/CourseCompletionViewModelTests.cs

```diff
@@ -0,0 +1,384 @@
+
+            // When
+            var courseCompletionViewModel = new CourseCompletionViewModel(expectedCourseCompletion);
+
+            // Then
+            courseCompletionViewModel.CustomisationId.Should().Be(customisationId);
+        }
+
+        [Test]
+        public void CourseCompletion_should_have_courseTitle()
+        {
+            // Given
+            const string applicationName = "Application Name";
+            const string customisationName = "Customisation Name";
+            var expectedCourseCompletion = CourseCompletionHelper.CreateDefaultCourseCompletion(
+                applicationName: applicationName,
+                customisationName: customisationName
+            );
+
+            // When
+            var courseCompletionViewModel = new CourseCompletionViewModel(expectedCourseCompletion);
+
+            // Then
+            var courseTitle = $"{applicationName} - {customisationName}";
+            courseCompletionViewModel.CourseTitle.Should().BeEquivalentTo(courseTitle);
+        }
+
+        [Test]
+        public void CourseCompletion_should_have_isAssessed()
+        {
+            // Given
+            const bool isAssessed = true;
+            var expectedCourseCompletion = CourseCompletionHelper.CreateDefaultCourseCompletion(
+                isAssessed: isAssessed
+            );
+
+            // When
+            var courseCompletionViewModel = new CourseCompletionViewModel(expectedCourseCompletion);
+
+            // Then
+            courseCompletionViewModel.IsAssessed.Should().Be(isAssessed);
+        }
+
+        [Test]
+        public void CourseCompletion_should_have_diagnosticScore()
+        {
+            // Given
+            const int diagnosticScore = 42;
+            var expectedCourseCompletion = CourseCompletionHelper.CreateDefaultCourseCompletion(
+                diagnosticScore: diagnosticScore
+            );
+
+            // When
+            var courseCompletionViewModel = new CourseCompletionViewModel(expectedCourseCompletion);
+
+            // Then
+            courseCompletionViewModel.DiagnosticScore.Should().Be(diagnosticScore);
+        }
+
+        [Test]
+        public void CourseCompletion_should_have_diagnosticAttempts()
+        {
+            // Given
+            const int diagnosticAttempts = 47;
+            var expectedCourseCompletion = CourseCompletionHelper.CreateDefaultCourseCompletion(
+                diagnosticAttempts: diagnosticAttempts
+            );
+
+            // When
+            var courseCompletionViewModel = new CourseCompletionViewModel(expectedCourseCompletion);
+
+            // Then
+            courseCompletionViewModel.DiagnosticAttempts.Should().Be(diagnosticAttempts);
+        }
+
+        [Test]
+        public void CourseCompletion_should_have_percentageTutorialsCompleted()
+        {
+            // Given
+            const int percentageTutorialsCompleted = 55;

```

Comment:
Is any rounding necessary here? 🙂

Reply:
This is getting truncated in the SQL query at the moment, and I should be rounding it, thanks for catching this! Should I round it in the SQL query, or the view model? 🙂

Reply:
View model would be more consistent with what we're doing on other pages as far as I remember 🙂

Reply:
Done in 6999d38 🙂